### DISCUSSION TITLE
Seeking Advice on Utilizing Fooocus on a Generated Model without Altering Branding

### DISCUSSION THREAD

**Discussion by massaro1994** [lllyasviel/Fooocus#2618] [General]
I hope this message finds you well. I am reaching out to seek some advice and guidance on utilizing Fooocus effectively in my current project. I am working on producing t-shirts featuring a specific brand logo, and I'm using a model generated by the same program. However, I'm encountering challenges in ensuring that Fooocus doesn't inadvertently modify the branding on the t-shirt during the rendering process.

Does anyone here have experience or suggestions on how to tackle this issue? I've experimented with various settings and techniques, but I haven't quite achieved the desired result yet. Any insights, tips, or best practices you could share would be greatly appreciated.

Thank you in advance for your assistance. Looking forward to your responses.

Best regards, Enrico

**Comment by mashb1t** [comment]
code for diffuse2choose by Amazon sadly hasn't been published yet, but if you really REALLY want this you can re-implement their paper.
see https://github.com/lllyasviel/Fooocus/discussions/2036#discussioncomment-8312113

There also is a similar project from Alibaba, but same story, code has not been published (yet).

Overlaying real products / using masks for products seem to be the currently best available optikn, maybe combined with differential diffusion mask inpainting to blend the different images indistinguishable from a photo. DifferentialDiffusion currently is only supported by A1111/Forge and Comfy, not yet Fooocus (see open issues as feature request).

Hope this helps!
